### PR TITLE
[OKU-26] FEAT - See More Reviews Page 

### PR DESCRIPTION
**Pull request by twentymurial33**
## Change Type
*Put an `x` in all the boxes that apply
- [] Bugfix (non-breaking change which fixes an issue)
- [X] Feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to change)

## Linear issue link
https://linear.app/oku-no-kumo/issue/OKU-28/see-more-reviews-page

## Before & after screenshots or video recordings (Before & After)
Screen Shot 2023-04-19 at 10 07 54 PM

### CODE REVIEW

File: project_app/pages/Reviews.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import CardComponent from "../components/CardComponent";
+import styled from "styled-components";
+import Button from "@mui/material/Button";
+import MenuItem from '@mui/material/MenuItem';
+import{Container,HeaderWrapper,ReviewBox,StyledLinks,NavBarButtons} from "./style";
```
Comment (idiglove) [inline project_app/pages/Reviews.tsx:5]: these are not prettified?

File: project_app/pages/Reviews.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+left:40%;
+
+h1{
+  font-size: 1.25rem;
+    color: rgb(255, 255, 255);
+    margin-right: 0.5rem;
+    z-index: 1;
+}
+
+  `
```
Comment (idiglove) [inline project_app/pages/Reviews.tsx:114]: put in a separate file

File: project_app/pages/Reviews.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+          </StyledLinks>
+      </Wrapper>
+     </HeaderWrapper>
+    <Container >
+      <ReviewBox>
+         <button>Create a Review</button> 
+          <CardComponent />
+      </ReviewBox>
+        </Container>
+  </>
```
Comment (idiglove) [inline project_app/pages/Reviews.tsx:80]: use the common Header that you made for this

**Review by idiglove — CHANGES_REQUESTED** [review]

File: project_app/pages/Reviews.tsx (hunk 3 above)
Comment (twentymurial33) [reply project_app/pages/Reviews.tsx:80]: imported some components as the style is a slightly different from the 1st one

File: project_app/pages/Reviews.tsx (hunk 2 above)
Comment (twentymurial33) [reply project_app/pages/Reviews.tsx:114]: Moved to style.tsx

**Review by idiglove — APPROVED** [review]
it's ok for now

**Pull request merged**